plugins/DevicesDetection/DevicesDetection.php

```diff
@@ -23,7 +23,6 @@ public function registerEvents()
     {
         return array(
             'Live.getAllVisitorDetails' => 'extendVisitorDetails',
-            'Request.getRenamedModuleAndAction' => 'renameUserSettingsModuleAndAction',
```

Comment:
Not sure why it was marked that way, as IMO there is no value to removing these calls. there is no code complexity involved for us, but as a result we would  force people to change some URLs deep into their app or product... which would be time consuming for them